ClientPlugin/MyCameraBlockPatch.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using HarmonyLib;
+using Sandbox.Game.Entities;
+using Sandbox.Game.Entities.Cube;
+using Sandbox.Game.Gui;
+using VRage.Utils;
+
+namespace ClientPlugin
+{
+    [HarmonyPatch]
+    [SuppressMessage("ReSharper", "UnusedType.Global")]
+    [SuppressMessage("ReSharper", "UnusedMember.Global")]
+    public class MyCameraBlockPatch
+    {
+        private static MethodBase TargetMethod()
```

Comment:
TargetMethod is not required here, since the `MyCameraBlock` type is public. You can use the normal `[HarmonyPatch(typeof(MyCameraBlock))]` on the class and `[HarmonyPatch("CreateTerminalControls")]` on the patch method.